### DISCUSSION TITLE
Manual Track ID Assignment for Top-Down Model Results

### DISCUSSION THREAD

**Discussion by neugun** [talmolab/sleap#1924] [Help!]
I am tracking two animals (one always on the left and the other on the right). When I use the top-down model (without the ID option, as the top-down ID model performs poorly in predictions), I don't get any tracks in the output. Is there a way to export the results for both animals and manually assign track IDs to the points?

![image](https://github.com/user-attachments/assets/ff0077ea-566e-47b8-9788-411116cba773)

**Comment by roomrys** [comment]
Hello @neugun,

When you run inference, what are you setting for the tracking parameters? I would recommend:
```
Tracker: simple
Max number of tracks: 2 (number of animals)
Connect Single Track Breaks: true
```
with the rest of the parameters set to default values.

I could write you a script to manually set the `Track`s afterwards if you are not having any luck with the temporal tracker (with parameters as specified above).

Let us know if you are still having trouble.

Thanks,
Liezl

**Comment by roomrys** [comment]
I am closing this discussion as we are following up in:
- #1925
